src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-seats/lancie-admin-seats.html

```diff
@@ -0,0 +1,430 @@
+      }
+
+      #listbox {
+        overflow-y: auto;
+        max-height: 40vh;
+      }
+
+      .seats {
+        display: flex;
+      }
+
+      .team-member {
+        cursor: pointer;
+      }
+    </style>
+
+    <lancie-ajax auto-fire id="ajaxGetAllSeats" refurl="seats" on-lancie-ajax="onGetAllSeats"></lancie-ajax>
+    <lancie-ajax auto-fire id="ajaxGetTeamTickets" refurl="tickets/teammembers" on-lancie-ajax="onGetTeamTickets"></lancie-ajax>
+    <lancie-ajax id="ajaxReserveSeat" method="POST" on-lancie-ajax="onReserveSeat"></lancie-ajax>
+    <lancie-ajax id="ajaxAddSeatRow" method="POST" body="[[getSeatGroupDTOBody(seatGroup,seatAmount)]]" on-lancie-ajax="onAddSeatRow" refurl = "seats"></lancie-ajax>
+    <lancie-ajax id="ajaxRemoveSeats" method="DELETE" body="[[getSeatGroupDTOBody(removeGroup,removeAmount)]]" on-lancie-ajax="onRemoveSeats" refurl = "seats"></lancie-ajax>
+    <lancie-ajax id="ajaxLockSeat" method="POST" body="[[lockBoolean]]" on-lancie-ajax="onLockSeat"></lancie-ajax>
+    <lancie-ajax id="ajaxLockGroup" method="POST" body="[[lockBoolean]]" on-lancie-ajax="onLockGroup"></lancie-ajax>
+
+    <paper-input label="Search user" value="{{filterUser}}"></paper-input>
+    <div class="seatmapwrapper">
+      <div class="seatmap">
+        <div class="seats">
+          <template is="dom-repeat" items='[[seatGroups]]'>
+            <div class="seatmap-row">
+              <div class="grouplabel" on-tap="openLockGroupDialog" data-item$="[[item]]">[[item]]</div>
+              <div class="seats-group">
+                <template is="dom-repeat" items="[[_getSeatGroup(seats, item)]]">
+                  <lancie-admin-seat tickets="[[tickets]]" filter="[[filterUser]]" seat="[[item]]" user="[[user]]" fixed="[[fixed]]"></lancie-admin-seat>
+                </template>
+              </div>
+            </div>
+          </template>
+          <div class="seatmap-row">
+            <div class="add-seat" on-tap="openAddSeatRowDialog">+</div>
+            <div class="add-seat" style="margin-top: 10px;" on-tap="openSeatRemoveDialog">-</div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <lancie-dialog id="seatDialog">
+      <h2>Seat reservation</h2>
+      <div class="dialog-content">
+        <lancie-error id="error"></lancie-error>
+        <div hidden$="[[_checkSeatTaken(seatToEdit)]]">
+          <h4>Seat currently taken by:</h4>
+          <p>[[seatToEdit.ticket.owner.profile.displayName]]</p>
+          
+        </div>
+        <div hidden$="[[!_checkSeatTaken(seatToEdit)]]">
+          <p>Seat is still available</p>
+        </div>
+      </div>
+      <div class="dialog-actions">
+        <paper-button on-tap="lockSeat">Toggle lock</paper-button>
+        <paper-button dialog-dismiss>Close</paper-button>
+      </div>
+    </lancie-dialog>
+
+    <lancie-dialog id=seatRowDialog>
+        <h2>Add a row of seats</h2>
+        <div class="dialog-content">
+          <lancie-error id="error2"></lancie-error>
```

Comment:
Let's not do this, give the element the correct name. `seatRowAddError` could be a suitable option here.

Reply:
This error was not used at all, hence I removed it.